KugleFirmware/KugleFirmware.xml

```diff
@@ -1,9 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do not commit this file change. Leave out these changes to the Eclipse configuration files.